docker/integration-tests/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM packs/cf
```

Comment:
This Dockerfile seems to be a duplicate of [this one in eirini-staging](https://github.com/cloudfoundry-incubator/eirini-staging/blob/master/image/integration-tests/Dockerfile). Can you verify this and remove if so?